webtau-core/src/main/java/com/twosigma/webtau/Ddjt.java

```diff
@@ -46,18 +47,18 @@
  * Convenient class for a single static * imports
  */
 public class Ddjt {
-    public static final TableDataUnderscoreOrPlaceholder __ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________________________________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-    public static final TableDataUnderscoreOrPlaceholder ________________________________________________________________________________________________ = TableDataUnderscoreOrPlaceholder.INSTANCE;
-
     public static final TableDataCellValueGenFunctions cell = new TableDataCellValueGenFunctions();
 
+    public static final TableDataCellValueGenerator<?> __ = TableDataCellAbove.generator;
```

Comment:
Not sure how I feel about the _____'s.  When I see that in the table, I don't immediately think that means value from above.  To me they indicate absence of value.

In written form, typically you'd do `-"-` but obviously that's not possible here so while I don't really like this I'm not sure I can think of a better alternative 😞 

Having said that, do we really need a shortcut?

Reply:
@tsiq-karold to me it felt non ambiguous, but if you feel that way there is a pretty good chance that it is not clear for others either. 
The main rationale for `_____` is it sort of let you see through important values only and is easier to glance over than `cell.above`. 

@tsiq-clemens what is your first reaction on `____`?

Pulling @markroth8 for additional opinion.

Reply:
Are we not using `_____` to also separate headers from data rows?  I vaguely recall a PR for that.

Reply:
There is still code to handle this.

Reply:
I understand that, I was referring to the fact that the underscores now mean two different things though I guess the context will generally make it clear which it is.

Reply:
Got a piece of a feedback. Immediate reaction is "value is empty", but with documentation it makes sense.